.github/CHANGELOG.md

```diff
@@ -50,6 +64,29 @@
     Hamiltonians, respectively, and apply the corresponding QAOA cost and mixer layers
     to the quantum circuit
 
+  For example, using PennyLane to construct and solve a MaxCut problem with QAOA:
+
+  ```python
+  wires = range(3)
+  graph = Graph([(0, 1), (1, 2), (2, 0)])
+  cost_h, mixer_h = qaoa.maxcut(graph)
+
+  def qaoa_layer(gamma, alpha):
+      qaoa.cost_layer(gamma, cost_h)
+      qaoa.mixer_layer(alpha, mixer_h)
+
+  def antatz(params, **kwargs):
```

Comment:
```suggestion
  def ansatz(params):
```

Reply:
Unfortunately this is requirement as @Lucaman99 pointed out. We have on our todo list to refactor the `map` function to remove this signature requirement.